Update information-barriers.md (#2500)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/information-barriers.md

![Example of segments in an organization](media/info-barriers-segments-example.png)

> With SharePoint information barriers, a SharePoint or global admin can associate segments to a site to prevent the site from being shared with or accessed by users outside the segments. Up to 100 compatible segments can be associated with a site. The segments are associated at the site level (previously called site collection level). Microsoft 365 group connected to the site is also associated with the site's segment.

Reviewer: ```suggestion
With SharePoint information barriers, a SharePoint or global admin can associate segments to a site to prevent the site from being shared with or accessed by users outside the segments. Up to 100 compatible segments can be associated with a site. The segments are associated at the site level (previously called site collection level). Microsoft 365 groups connected to the site is also associated with the site's segment.
```

In the above example, the HR segment is compatible with both Sales and Research. However, because the Sales and Research segments are incompatible, they can't be associated with the same site. 

## Prerequisites

- [Make sure you meet the licensing requirements for information barriers](/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance#information-barriers).
- Create segments and define the users in each. Create policies that allow or block communication between the segments, and then set them to active. For info, see [Define policies for information barriers](/office365/securitycompliance/information-barriers-policies).

> - [Complete the form](https://forms.microsoft.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR_mCQfArPllNo4NRhexFZxRUNEZCTTZLVVJHOEJOVVlCTUdJNDVERko5Qy4u) to enable SharePoint and OneDrive information barriers in your organization. Please note - SharePoint and OneDrive information barriers can only be enabled in your tenant if you already have atleast one active information barriers policy. 

Reviewer: ```suggestion
- [Complete the form](https://forms.microsoft.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR_mCQfArPllNo4NRhexFZxRUNEZCTTZLVVJHOEJOVVlCTUdJNDVERko5Qy4u) to enable SharePoint and OneDrive information barriers in your organization. Please note that SharePoint and OneDrive information barriers can only be enabled in your tenant if you already have at least one active information barriers policy. 
```

> - After SharePoint and OneDrive information barriers is enabled in your tenant, wait 24 hours for the changes to propagate through your environment. 

Reviewer: ```suggestion
- After SharePoint and OneDrive information barriers are enabled in your tenant, wait 24 hours for the changes to propagate through your environment. 
```
